DAL/user.DAL.js

```diff
@@ -0,0 +1,22 @@
+const User = require("./../modules/user.module");
+
+const addNewUser = async (user) => {
+  const newUser = User.create(user);
+  return newUser;
+};
+
+const getUserByEmail = async (req, res) => {
+  const foundUser = User.findOne({ email: req.body.email });
+  return foundUser;
+};
+
+const getUserById = async (req, res) => {
+  const foundUser = User.findOne({ id: req.body._id });
+  return foundUser;
+};
+
+module.exports = {
```

Comment:
`addNewUser: addNewUser,...`
do not duplicate key and value names while they are the same. 
use ES6^ rules

Reply:
DONE